pcf/test/particle/aws/test_particles.py

```diff
@@ -0,0 +1,54 @@
+import moto
+import json
+import pytest
+import boto3
+import os
+import placebo
+from pcf.core import State
+from pcf.core import particle_flavor_scanner
+from contextlib import ExitStack
+
+from pcf.particle.aws.route53 import hosted_zone
+from pcf.particle.aws.sqs.sqs_queue import SQSQueue
+from pcf.particle.aws.cloudfront.cloudfront import CloudFront
+
+with open("testdata.json") as f:
```

Comment:
E   FileNotFoundError: [Errno 2] No such file or directory: 'testdata.json'

since pytest is run for different directory. use `os` to get the right location